trackthenews/__init__.py

```diff
@@ -1 +1 @@
-from .core import *
+from .core import main  # noqa: F401
```

Comment:
There's no need for a wildcard import as far as I can tell; `setup.py` only specifies `main` as an entry point.